docs/sources/mimir/manage/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -14,257 +14,30 @@ You can collect logs and metrics from a Mimir or GEM cluster. To set up dashboar
 see [Installing Grafana Mimir dashboards and alerts]({{< relref "./installing-dashboards-and-alerts" >}})
 or [Grafana Cloud: Self-hosted Grafana Mimir integration](/docs/grafana-cloud/monitor-infrastructure/integrations/integration-reference/integration-mimir/).
 
-It is easier and best to monitor a cluster if it was installed via the Grafana Mimir Helm chart.
-For more information, see the [documentation for the Grafana Mimir Helm chart](/docs/helm-charts/mimir-distributed/latest/).
+## Install Grafana Mimir using the Helm chart
 
-It is also possible to use this integration if Mimir was deployed another way.
-For more information, see [Collect metrics and logs without the Helm chart]({{< relref "#collect-metrics-and-logs-without-the-helm-chart" >}}).
+As a best practice, use the Grafana Mimir Helm chart to install a Mimir or GEM cluster. When you deploy the Helm chart, ensure that you've enabled `ServiceMonitor` objects. For more information, refer to the [documentation for the Grafana Mimir Helm chart](/docs/helm-charts/mimir-distributed/latest/).
```

Comment:
@eamonryan please verify that I captured this correctly and add any additional information we want to include.

Reply:
Looks good!